src/NHibernate.Test/SqlTest/Custom/CustomSQLSupportTest.cs

```diff
@@ -64,7 +65,13 @@ public void HandSQL()
 			IEnumerator iter = s.GetNamedQuery("allOrganizationsWithEmployees").List().GetEnumerator();
 			Assert.IsTrue(iter.MoveNext());
 			Organization o = (Organization)iter.Current;
-			Assert.AreEqual(o.Employments.Count, 3);
+
+			Assert.AreEqual(3, o.Employments.Count);
+
+			foreach (Employment e in o.Employments)
+			{
+				s.Delete(e);
+			}
 
 			foreach (Employment e in o.Employments)
 			{
```

Comment:
Failed copy paste? (Why the duplication of these lines of code?)

Reply:
Tool failed probably (this was done by a tool)